lib/chat_api/accounts.ex

```diff
@@ -21,6 +22,7 @@ defmodule ChatApi.Accounts do
     Repo.all(Account)
   end
 
+  @spec get_account!(integer) :: Account.t() | nil
```

Comment:
we actually use UUIDs instead of integers for IDs for all our tables (except for the users table, which is a bit of tech debt) -- sorry for the confusion with the docstring below!

Reply:
also instead of `nil`, should this be `Ecto.NoResultsError`?

Reply:
It should be just `@spec get_account!(binary()) :: Account.t()` it either return `Account` or throw exception :)

Reply:
makes sense! 👍